lib/rules/headers/dl.js

```diff
@@ -190,6 +190,20 @@ exports.check = function (sr, done) {
         else sr.error(self, "supersedes-syntax");
     }
 
+    if (dts.Implementation) {
+        var linkImple = dts.Implementation.dd.querySelector("a") && dts.Implementation.dd.querySelector("a").href;
+        if (linkImple && !linkImple.match(/^https:\/\//)) {
```

Comment:
if you only want to check if a string starts with another string, you can use [`startsWith`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/startsWith). It's also best to [`lowercase`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase) the string first to make sure you test strings like 'Https://'...

The comment also applies in the other changes.